package.json

```
@@ -8,8 +8,10 @@
     "ci:packages": "lerna exec -- npm ci",
     "clear_styled_components": "for d in ./packages/*/*; do rm -rf $d/node_modules/styled-components; done; echo \"Cleared nested packages/**/node_modules/styled-components\"",
     "deploy-storybook": "build-storybook -o storybook_dist && gh-pages -d storybook_dist",
+    "force-package-lock-https": "find . -type f -name \"package-lock.json\" | grep -v node_modules | xargs -I % sh -c 'echo \"Replacing http->https in %\"; sed -i \"\" -e \"s/http:\\/\\//https:\\/\\//g\" %'",
     "install:packages": "lerna exec -- npm install",
     "postinstall": "npm run ci:packages",
+    "postshrinkwrap": "if test -z $JENKINS_HOME; then \n npm run force-package-lock-https \n fi",
```

```suggestion
    "postshrinkwrap": "if test -z $CI; then \n npm run force-package-lock-https \n fi",
```
Uses https://github.com/BBC-News/psammead/blob/latest/Jenkinsfile#L22